tests/unit/viahtml/app_test.py

```diff
@@ -0,0 +1,90 @@
+import os
+from unittest.mock import patch
+
+import pytest
+from h_matchers import Any
+from pytest import param
+from pywb.apps.frontendapp import FrontEndApp
+from pywb.apps.rewriterapp import RewriterApp
+
+from viahtml.app import Application
+
+
+# For reasons I find absolutely mysterious, everytime we create our app
+# coverage can no longer be detected on lines following it. We absolutely get
+# there but for some reason it doesn't show in the report?
+class TestApplicationCreate:  # pragma: no cover
+    def test_it_returns_an_app(self):
+        app = Application.create()
+
+        assert isinstance(app, FrontEndApp)
+
+    @patch("viahtml.app.apply_pre_app_hooks", autospec=True)
```

Comment:
We should copy-paste [the `patch` fixture](https://github.com/hypothesis/lms/blob/869911e7093b6cabfca41e9a64828725905f8f38/tests/conftest.py#L62-L74) from our other apps and use that. It's easier to use, integrates with pytest fixtures (class, module or package-level autousing, overriding, ...), ensures that `autospec` is always used, and is consistent with the rest of our apps

Reply:
We could, and possibly should elsewhere. I also think we should be familiar with the tools available in the libraries we use, rather than using our own re-inventions 100% of the time.

For the reasons you mentioned we should do something like that when we intend to re-use the fixture.

I think we should use this pattern when we need a single fixture once in one place. It's much more concise, and keeps everything relevant near the test in question (no scrolling).